utf8.c

```diff
@@ -1600,7 +1600,7 @@ Perl_utf8_to_uv_msgs_helper_(const U8 * const s0,
     SSize_t curlen = send - s0;
     U32 possible_problems;  /* A bit is set here for each potential problem
                                found as we go along */
-    UV uv;
+    UV uv = UV_MAX;
```

Comment:
Note:  initializing `UV uv` to `0` here would also eliminate the `uninitialized value` warning (without causing test suite failures).  Reviewers should indicate whether that's a better choice or not.

Reply:
Initializing to `0` seems sound to me, please.

I'm not 100% sure of the logic flow in this function, so it might be that `uv` always gets assigned to in practice, but since `uv` is supposed to contain packed flags, starting with it set to zero (no flags yet added) at the start of the function makes sense.

Reply:
@richardleach, `uv` is not flags, it is the code point the input UTF-8 represents, expressed as a UV

Reply:
There is more work to be done on the draft PR.  If you need this out in the imminent dot release, it should be initialized to zero.

Reply:
> There is more work to be done on the draft PR. If you need this out in the imminent dot release, it should be initialized to zero.

Thanks for looking at this.  I will update the branch, let it smoke-test for about a day, then merge it into blead (hopefully) in advance of the monthly release.